src/components/PokemonList.jsx

```diff
@@ -32,6 +41,17 @@ const PokemonList = ({
         onChange={handleQueryChange}
         placeholder=" Search your Pokemon by name..."
       />
+      <select name="list-choice" id="list-choice" onChange={handleList}>
+        <option>1-99</option>
```

Comment:
`<option value="0">` would be **much** better !

Reply:
👍